docs/ref/contrib/gis/serializers.txt

```diff
@@ -71,3 +71,11 @@ Would output::
         }
       ]
     }
+
+When the ``fields`` parameter is not specified, the ``geojson`` serializer puts
```

Comment:
this formulation might be a tad more readable: "the geojson serializer adds a 'pk' key to the properties dictionary with the primary key of the object as the value."